cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.5",
```

Comment:
See discussion here: https://github.com/kubernetes/kubernetes/pull/32404#r78450928

It's safe